src/journal/views.py

```diff
@@ -1319,6 +1320,35 @@ def publish_article_check(request, article_id):
         return HttpResponse(json.dumps({'action': 'ok', 'id': value}), content_type="application/json")
 
 
+@staff_member_required
+def view_jats_stub(request, article_id):
+    """
+    A staff view for previewing, and validating JATS metadata. This view is only
+    accessible when DEBUG is set to True in settings.py.
+    :param request: HttpRequest object
+    :param article_id: Article object PK
+    """
+    article = get_object_or_404(
+        submission_models.Article,
+        pk=article_id,
+        journal=request.journal,
+    )
+    context = {
+        'article': article,
+        'include_declaration': True,
+    }
+    xml_output = render_to_string(
+        'common/encoding/article_jats_1_2.xml',
+        context,
+    )
+    valid, error_message = xml_validation.validate_jats_with_remote_dtd(xml_output)
+
+    if not valid:
+        error_comment = f"<!-- JATS Validation Error: {error_message} -->\n"
+        xml_output = xml_output + error_comment
```

Comment:
Adding a comment at the top of the XML document will make it not XML valid and some browsers might display an error. if we need to add a comment, it should be after the initial XML declaration.